server/controllers/handleQuestItemClicked.js

```diff
@@ -41,21 +41,20 @@ export const handleQuestItemClicked = async (req, res) => {
 
     const itemsCollectedByUser = world.dataObject?.keyAssets?.[keyAssetId]?.itemsCollectedByUser;
     const numberAllowedToCollect = keyAsset.dataObject?.numberAllowedToCollect;
-    let numberCollected = 0;
+    let numberCollectedToday = 1;
 
     if (
       itemsCollectedByUser &&
       itemsCollectedByUser[profileId] &&
-      itemsCollectedByUser[profileId][dateKey] &&
-      itemsCollectedByUser[profileId][dateKey].count >= numberAllowedToCollect
+      itemsCollectedByUser[profileId].totalCollectedToday >= numberAllowedToCollect
     ) {
       console.log(`Visitor has already collected ${numberAllowedToCollect} quest items today.`);
```

Comment:
I'm not It's good to have console.logs in the middle of the code, because it will make cloudwatch costly. I mean, we could keep the logs there, but this log doesn't help much to identify bugs